torch/_tensor_docs.py

```diff
@@ -2460,6 +2460,30 @@ def callable(a, b) -> number
 See :func:`torch.topk`
 """)
 
+add_docstr_all('to_sparse',
+               r"""
+to_sparse(sparseDims) -> Tensor
+Returns a sparse copy of the tensor.  PyTorch supports sparse tensors in
+:ref:`coordinate format <sparse-docs>`.
+Args:
+    sparseDims (int, optional): the number of sparse dimensions to include in the new sparse tensor
+Example::
+    >>> d = torch.tensor([[0, 0, 0], [9, 0, 10], [0, 0, 0]])
+    >>> d
+    tensor([[ 0,  0,  0],
+            [ 9,  0, 10],
+            [ 0,  0,  0]])
+    >>> d.to_sparse()
+    tensor(indices=tensor([[1, 1],
+                           [0, 2]]),
+        values=tensor([ 9, 10]),
```

Comment:
please align `values` with indices, sorry I should've checked this at my previous pass

Reply:
no worries, good catch